modules/installation-guide/examples/checluster-properties.adoc

```diff
@@ -45,125 +45,127 @@ pluginRegistryUrl: Public URL of the plugin registry that serves sample ready-to
+serverExposureStrategy: Sets the server and workspaces exposure type. Possible values are `multi-host`, `single-host`, `default-host`. Defaults to `multi-host`, which creates a separate ingress, or OpenShift routes, for every required endpoint. `single-host` makes Che exposed on a single host name with workspaces exposed on subpaths. Read the docs to learn about the limitations of this approach. Also consult the `singleHostExposureType` property to further configure how the Operator and the Che server make that happen on Kubernetes. `default-host` exposes the Che server on the host of the cluster. Read the docs to learn about the limitations of this approach.
+serverMemoryLimit: Overrides the memory limit used in the Che server deployment. Defaults to 1Gi.
+serverMemoryRequest: Overrides the memory request used in the Che server deployment. Defaults to 512Mi.
+serverTrustStoreConfigMapName: Name of the ConfigMap with public certificates to add to Java trust store of the Che server. This is often required when adding the OpenShift OAuth provider, which has HTTPS endpoint signed with self-signed cert. The Che server must be aware of its CA cert to be able to request it. This is disabled by default.
 singleHostGatewayConfigMapLabels: The labels that need to be present in the ConfigMaps representing the gateway configuration.
 singleHostGatewayConfigSidecarImage: The image used for the gateway sidecar that provides configuration to the gateway. Omit it or leave it empty to use the default container image provided by the Operator.
 singleHostGatewayImage: The image used for the gateway in the single host mode. Omit it or leave it empty to use the default container image provided by the Operator.
-tlsSupport: Deprecated. Instructs the Operator to deploy {prod-short} in TLS mode. This is enabled by default. Disabling TLS sometimes cause malfunction of some {prod-short} components.
+tlsSupport: Deprecated. Instructs the Operator to deploy Che in TLS mode. This is enabled by default. Disabling TLS sometimes cause malfunction of some Che components.
 useInternalClusterSVCNames: Use internal cluster SVC names to communicate between components to speed up the traffic and avoid proxy issues. The default value is `true`.
-workspaceNamespaceDefault: Defines default {platforms-namespace} in which user's workspaces are created for a case when a user does not override it. It's possible to use `<username>`, `<userid>` and `<workspaceid>` placeholders, such as `che-workspace-<username>`. In that case, a new {platforms-namespace} will be created for each user or workspace.
-:===
+workspaceNamespaceDefault: Defines Kubernetes default namespace in which user's workspaces are created for a case when a user does not override it. It's possible to use `<username>`, `<userid>` and `<workspaceid>` placeholders, such as che-workspace-<username>. In that case, a new namespace will be created for each user or workspace.
```

Comment:
as you changed it above, this could be EITHER a Kubernetes namespace or an OpenShift project.